src/utils/navigate.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Navigates to the specified URL, including Account ID param if available.
+ * This function may or may not return. The caller should account for this!
+ */
+export function navigate (url: string): boolean {
+  try {
+    // get account id and set in params
+    const accountId = sessionStorage.getItem('ACCOUNT_ID')
+    if (accountId) {
+      if (url.includes('?')) {
+        url += `&accountid=${accountId}`
+      } else {
+        url += `?accountid=${accountId}`
+      }
+    }
+    // assume URL is always reachable
+    window.location.assign(url)
+
+    return true
+  } catch (error) {
+    console.log('Error navigating =', error) // eslint-disable-line no-console
+
+    return false
+  }
+}
```

Comment:
There's a sexier way to parse the URL using `new URL(xxx)` and to get/set the parameters using `new URLSearchParams(yyy)` but it doesn't work consistently in the unit tests, so I rolled my own code.